### PR TITLE
fix: updated CRD from apiextensions/v1beta1 to v1 (#5515)

### LINKED ISSUE #5515: Update CRDs to apiextensions.k8s.io/v1
# Summary

The current CRDs are using v1beta1 of the CustomResourceDefinition (https://github.com/argoproj/argo-cd/tree/master/manifests/crds). We should upgrade it to v1.

# Motivation

v1beta1 is deprecated since Kubernetes v1.16.

# Proposal

Upgrade to v1. The ArgoCD Helm chart CRDs have been updated https://github.com/argoproj/argo-helm/pull/545 or are in the process of being updated https://github.com/argoproj/argo-helm/pull/576.

I can definitely create a PR for this!

### PR DESCRIPTION
**Pull request by chgl**
Closes #5515 

Best viewed when disabling whitespace in diff: https://github.com/argoproj/argo-cd/pull/5516/files?diff=split&w=1

Note on DCO:

If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [x] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-cd/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this does not need to be in the release notes.
* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [x] I've included "Closes [ISSUE #]" or "Fixes [ISSUE #]" in the description to automatically close the associated issue.
* [ ] I've updated both the CLI and UI to expose my feature, or I plan to submit a second PR with them.
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [ ] Optional. My organization is added to USERS.md.
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo)
* [x] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/latest/developer-guide/ci/)).

### CODE REVIEW

**Comment by jannfis** [comment]
Hi @chgl, sorry for the late response to this PR, which is btw much appreciated.

Can you please rebase your PR to latest changes in master and re-run `make codegen`, then push changes back?

**Comment by chgl** [comment]
@jannfis just rebased. thank you!

File: manifests/install.yaml [hunk 1]
```diff
@@ -1,23 +1,12 @@
...
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   labels:
     app.kubernetes.io/name: applications.argoproj.io
     app.kubernetes.io/part-of: argocd
   name: applications.argoproj.io
 spec:
-  additionalPrinterColumns:
```
Comment (sbose78) [inline manifests/install.yaml:10]: Curious, why are we removing the printer columns?

Comment (chgl) [reply manifests/install.yaml:10]: They're still there! Just moved inside the `versions` array, see: https://github.com/argoproj/argo-cd/pull/5516/files/527dfbef15d9dbe99140b5baeeefeb8f6ed24b01?diff=split&w=1#diff-085adbc8bea5279ae0da31de54e660d34708509979e708a566d8e5f5339822d8R21

Comment (sbose78) [reply manifests/install.yaml:10]: Great, thanks!

**Review by jannfis — APPROVED** [review]
LGTM, thank you @chgl

**Pull request merged**